aospy/test/test_calc_basic.py

```diff
@@ -39,109 +42,122 @@ def _test_files_and_attrs(calc, dtype_out):
     _test_output_attrs(calc, dtype_out)
 
 
-_BASIC_TEST_PARAMS = {
-    'proj': example_proj,
-    'model': example_model,
-    'run': example_run,
-    'var': condensation_rain,
-    'date_range': (datetime.datetime(4, 1, 1),
-                   datetime.datetime(6, 12, 31)),
-    'intvl_in': 'monthly',
-    'dtype_in_time': 'ts'
+_2D_DATE_RANGES = {
+    'datetime': (datetime.datetime(4, 1, 1), datetime.datetime(6, 12, 31)),
+    'datetime64': (np.datetime64('0004-01-01'), np.datetime64('0006-12-31')),
+    'cftime': (cftime.DatetimeNoLeap(4, 1, 1),
```

Comment:
Do you think we're safe testing only against `DatetimeNoLeap`, or should we loop over the other calendars as well?  (Maybe should have been doing so irrespective of this PR.)

Reply:
At this level I think we're OK, so long as we have thorough test coverage for the lower-level functions.  I hesitate to add more types to these tests, because they are somewhat slow.